### ISSUE TITLE
Changing default tiddler location fails with error in Linux Mint

### ISSUE THREAD

**Issue by ibnishak** [Jermolene/TiddlyWiki5#2762]
Relevant portion of tiddlywiki.info

	"config": {
		"default-tiddler-location": "./tiddlers/myDefault"	
		}

Message from terminal: 

	Sync error while processing 'New Tiddler':
	Error: ENOENT: no such file or directory, scandir '/home/who/Cloud/Gdrive/TW/notes/tiddlers/myDefault'

I tried manually creating the folder "myDefault" under tiddlers. As soon as I saved the "New Tiddler", the server deleted that folder. Later I tried setting "myDefault" in various locations with the same outcome.

**OS**: Linux Mint 18
**Node**: v 6.9.1
**Tiddlywiki**: v 5.1.13

**Comment by Jermolene** [comment]
Hi @ibnishak 

> I tried manually creating the folder "myDefault" under tiddlers. As soon as I saved the "New Tiddler", the server deleted that folder. Later I tried setting "myDefault" in various locations with the same outcome.

Very strange.

I tried to duplicate the issue (on macOS) with the following steps:

```
mkdir ~/testwiki
cd ~/testwiki
tiddlywiki --init server
```

Then I edited `tiddlywiki.info` to add the config line above, and then ran the server with:

```
tiddlywiki --server
```

I then opened http://127.0.0.1:8080 in a browser and created a new tiddler. The new `.tid` file was created in `~/testwiki/tiddlers/myDefault`, as expected.

It might be interesting for you to try a similar minimal test case on your setup and see if you run into the same problem. It may just be a platform-specific issue about directory creation.
